src/Pages/ProductDetails/Components/GoodsDetails/Components/GoodsDescription/GoodsDescription.js

```diff
@@ -0,0 +1,78 @@
+import Taps from 'Pages/ProductDetails/Components/Taps/Taps';
+import React, { Component } from 'react';
```

Comment:
```suggestion
import React, { Component } from 'react';
import Taps from 'Pages/ProductDetails/Components/Taps/Taps';
```
import 순서..! 어떤 순서로 적용하는지 알고 계시죠?